### PR TITLE
Swapped RCT2_ADDRESS_*_PATH access with dynamic path making.

### PR DESCRIPTION
**Pull request by xiane**
- Swapped RCT2_ADDRESS_*_PATH access with dynamic path making. (game_path, saved_game_path, landscape_path, tracks_path, object_data_path)
- Not do it just RCT2_ADDRESS_SAVED_GAMES_PATH_2.
- Removed definition of that.

### CODE REVIEW

File: src/addresses.h [hunk 1]
```diff
@@ -93,12 +91,6 @@
 #define RCT2_ADDRESS_CONFIG_RESOLUTION_WIDTH		0x009AB4C2
 #define RCT2_ADDRESS_CONFIG_RESOLUTION_HEIGHT		0x009AB4C4
 
-#define RCT2_ADDRESS_APP_PATH_SLASH					0x009AB4D9
-#define RCT2_ADDRESS_SAVED_GAMES_PATH				0x009AB5DA
-#define RCT2_ADDRESS_SCENARIOS_PATH					0x009AB6E9
-#define RCT2_ADDRESS_LANDSCAPES_PATH				0x009AB7FB
-#define RCT2_ADDRESS_OBJECT_DATA_PATH				0x009AB90E
-#define RCT2_ADDRESS_TRACKS_PATH					0x009ABA1E
```
Comment (duncanspumpkin) [inline src/addresses.h]: Can you move all of these defines into the depreciated section rather than remove them.

**Comment by duncanspumpkin** [comment]
I'd also prefer it if you used an ENUM for the different fields and then had an array of the different folders.

File: src/addresses.h (hunk 1 above)
Comment (xiane) [inline src/addresses.h]: I moved it to the below section.

**Comment by Gymnasiast** [comment]
Thank you for your contribution.
We have decided not to merge this as it is, as the code has had incompatible changes since you created it (with more coming up) and because we want to do some things differently.

**Comment by xiane** [comment]
Okay, I understood. Thank you. :)

**Pull request closed without merging**